lib/Symbols.js

```diff
@@ -61,14 +63,53 @@ Symbols.getLastNonSymbolTerm = function(node, symbolName) {
   return null;
 };
 
-// Returns if `node` is a polynomial term with symbol `symbolName`
+// Iterates through a node and returns the denominator if it has a
+// symbolName in its denominator
+// e.g. 1/(2x) with `symbolName=x` would return 2x
+// e.g. 1/(x+2) with `symbolName=x` would return x+2
+// e.g. 1/(x+2) + (x+1)/(2x+3) with `symbolName=x` would return 2x+3
+Symbols.getLastDenominatorWithSymbolTerm = function(node, symbolName) {
+  // First check if the node itself has a symbol in the denominator
+  if (hasDenominatorSymbol(node, symbolName)) {
+    return node.args[1];
+  }
+  // Otherwise, it's a sum of terms. e.g. 1/x + 1(2+x)
+  // Look through the operands for a
+  // denominator term with `symbolName`
+  else if (Node.Type.isOperator(node) && node.op === '+') {
+    for (let i = node.args.length - 1; i >= 0 ; i--) {
+      const child = node.args[i];
+      if (hasDenominatorSymbol(child, symbolName)) {
+        return child.args[1];
+      }
+    }
+  }
+  return null;
+};
+
+// Returns if `node` is a term with symbol `symbolName`
 function isSymbolTerm(node, symbolName) {
   if (Node.PolynomialTerm.isPolynomialTerm(node)) {
     const polyTerm = new Node.PolynomialTerm(node);
     if (polyTerm.getSymbolName() === symbolName) {
       return true;
     }
   }
+  else if (hasDenominatorSymbol(node, symbolName)) {
+    return true;
+  }
+  return false;
+}
+
+// Return if `node` has a symbol in its denominator
+// e.g. true for 1/(2x)
+// e.g. false for 5x
+function hasDenominatorSymbol(node, symbolName) {
+  if (Node.Type.isOperator(node) && node.op === '/') {
+    const allSymbols = Symbols.getSymbolsInExpression(node);
```

Comment:
I think this will return true for `x/2` too, so should this line be calling `Symbols.getSymbolsInExpression` only on the denominator?